### DISCUSSION TITLE
How about list API for country and country name

### DISCUSSION THREAD

**Discussion by tommoholmes** [saleor/saleor#10236] [General]
I very difficult to add new address when save country name,
I'm always error,
how about to list all country name and user can select of country 
![Screen Shot 2022-07-26 at 10 27 15](https://user-images.githubusercontent.com/12724592/180916686-ac9eac29-2798-473b-93a2-9b48db3a0bc6.png)

**Comment by karolkielecki** [comment]
Selamat pagi @tommoholmes, `yogya` is not a country area. You need to type a full province name there. Please try with `Yogyakarta`. You can learn more here: https://github.com/saleor/saleor/discussions/9930

Indonesian address validation rules below:

```graphql
{
  "data": {
    "addressValidationRules": {
      "countryAreaType": "province",
      "cityType": "city",
      "cityAreaType": "suburb"
    }
  },
  "extensions": {
    "cost": {
      "requestedQueryCost": 1,
      "maximumAvailable": 50000
    }
  }
}
```

**Reply by tommoholmes** [reply]
thx for your quick reply me
what I mean is, list for country and provinces,
if we provide list api and displays it, I think no typo or incorrect again like me
